### PR TITLE
fix parent grab dropping and cullig flickering when flying at high velocities 

### PR DESCRIPTION
**Pull request by danteruiz**
fix parent grab dropping and cullig flickering when flying at high velocities, the final thing to fix for this ticket
fix parent grab dropping and cullig flickering when flying at high velocities

### CODE REVIEW

File: scripts/system/controllers/controllerModules/nearParentGrabEntity.js [hunk 1]
```diff
@@ -173,11 +173,7 @@ Script.include("/~/system/libraries/cloneEntityUtils.js");
                 this.lastUnequipCheckTime = now;
                 if (props.parentID === MyAvatar.SELF_ID) {
                     var sensorScaleFactor = MyAvatar.sensorToWorldScale;
-                    var handPosition = controllerData.controllerLocations[this.hand].position;
-                    var dist = distanceBetweenPointAndEntityBoundingBox(handPosition, props);
-                    var distance = Vec3.distance(props.position, handPosition);
-                    if ((dist > TEAR_AWAY_DISTANCE) ||
-                        (distance > NEAR_GRAB_RADIUS * sensorScaleFactor)) {
+                    if ((props.localPosition > TEAR_AWAY_DISTANCE * sensorScaleFactor)) {
```
Comment (SamGondelman) [inline scripts/system/controllers/controllerModules/nearParentGrabEntity.js]: Does this get updated by other people grabbing it?

Comment (sethalves) [reply scripts/system/controllers/controllerModules/nearParentGrabEntity.js]: This code may need to check to see if this is an action grab or a parenting grab and use the old version or the new version, accordingly.  An action grab wont have a useful localPosition.

Comment (danteruiz) [reply scripts/system/controllers/controllerModules/nearParentGrabEntity.js]: not sure.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11175/HighFidelity-Beta-PR13560-ae03dcb.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11175/HighFidelity-Beta-PR13560-ae03dcb.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11175/HighFidelity-Beta-Interface-PR13560-ae03dcb.dmg

**Comment by danteruiz** [comment]
Test plan (HMD)
- enable flying and advance movement
- spawn an entity
- grab the entity and fly around at high speeds.
- The entity should not be dropped unless you let go of the trigger
- also when flying around with the cube, the cube should not flicker viciously

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11175/HighFidelity-Beta-PR13560-ae03dcb.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11175/HighFidelity-Beta-Interface-PR13560-ae03dcb.exe

File: scripts/system/controllers/controllerModules/nearParentGrabEntity.js (hunk 1 above)
Comment (AndrewMeadows) [reply scripts/system/controllers/controllerModules/nearParentGrabEntity.js]: This does not look correct.  Is `props.localPosition` a 3D vector or a scalar?  The variable name suggests it is a 3D vector but it is being compared to a scalar.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11344/HighFidelity-Beta-PR13560-20a8275.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11344/HighFidelity-Beta-PR13560-20a8275.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11344/HighFidelity-Beta-Interface-PR13560-20a8275.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11344/HighFidelity-Beta-PR13560-20a8275.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11344/HighFidelity-Beta-Interface-PR13560-20a8275.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Pull request merged**